front/src/LayoutContainer.tsx

```diff
@@ -42,58 +43,70 @@ export default withRouter(function LayoutContainer({ history }) {
   }
 
   return (
-    <React.Fragment>
-      <Header isAuthenticated={isAuthenticated} />
-      <Route
-        exact
-        path="/"
-        render={() => <Home isAuthenticated={isAuthenticated} />}
-      />
-      <Route exact path="/cgu" component={WaitingComponent(Cgu)} />
-      <Route exact path="/partners" component={WaitingComponent(Partners)} />
-      <Route exact path="/login" component={WaitingComponent(Login)} />
-      <Route exact path="/invite" component={WaitingComponent(Invite)} />
-      <Route exact path="/signup" component={WaitingComponent(Signup)} />
-      <Route
-        exact
-        path="/signup/details"
-        component={WaitingComponent(WasteSelector)}
-      />
-      <Route
-        exact
-        path="/signup/activation"
-        component={WaitingComponent(SignupInfo)}
-      />
-      <Route
+    <Switch>
+      <PrivateRoute
         exact
-        path="/reset-password"
-        component={WaitingComponent(ResetPassword)}
+        path="/oauth2/authorize/dialog"
```

Comment:
J'ai du le louper, mais qu'est ce qui trigger l'ouverture de cette Dialog ?

Reply:
C'est la boite de dialogue sur laquelle va rediriger le bouton "Se connecter avec TD" des applications tierces pour amorcer le protocole OAuth2.

Reply:
Aaah yes of course, ca s'ouvre depuis les applis tierces, thanks 👍